src/bc_handling/src/bc_handling_base.C

```diff
@@ -321,7 +321,7 @@ namespace GRINS
                                       const std::string& bc_vars,
                                       const std::string& bc_value,
 				      const GetPot& input )
-  {
+  {    
```

Comment:
Arg, dammit, thanks for catching that.

@vikramvgarg Turn on "remove trailing whitespace on save" in your text editor!
http://emacswiki.org/emacs/DeletingWhitespace
http://vim.wikia.com/wiki/Remove_unwanted_spaces